### PR TITLE
PRESS2-1376 Add/sitegen socialmedia

### PR DESCRIPTION
**Pull request by girish-lokapure**
Screenshot Capture - 2023-11-15 - 18-37-03
Screenshot Capture - 2023-11-24 - 14-59-26

### CODE REVIEW

File: src/OnboardingSPA/steps/SiteGen/SocialMedia/index.js [hunk 1]
```diff
@@ -22,12 +24,40 @@ const SiteGenSiteSocialMedia = () => {
...
+						{ content.heading }
+					</p>
+				</div>
+				<div className="nfd-onboarding-step--site-gen__social-media__contain ">
+					<div className="nfd-onboarding-step--site-gen__social-media__contain__containleft ">
+						{ content.facebookTitle }
+						<p>{ content.facebookdesc }</p>
+					</div>
+					<div className="nfd-onboarding-step--site-gen__social-media__contain__containright ">
+						<button className="nfd-onboarding-step--site-gen__social-media__contain__containright__button ">
```
Comment (officiallygod) [inline src/OnboardingSPA/steps/SiteGen/SocialMedia/index.js:47]: I think the icon is supposed to be white? 
Screenshot 2023-11-20 at 1 18 49 PM

File: src/OnboardingSPA/steps/SiteGen/SocialMedia/stylesheet.scss [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+						}
+					}
+				}
+
+			}
+
+			&__contain {
+				width: 600px;
+				margin: 16px;
+				border: 1.25px dashed var(--nfd-onboarding-site-logo-border);
```
Comment (officiallygod) [inline src/OnboardingSPA/steps/SiteGen/SocialMedia/stylesheet.scss:57]: Should this be dotted?
Screenshot 2023-11-20 at 1 18 49 PM

File: src/OnboardingSPA/steps/SiteGen/SocialMedia/contents.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+	return {
+		heading: __(
+			'Do you want to include any content from Facebook?',
+			'wp-module-onboarding'
+		),
+		facebookTitle: __(
+			'Connect a Facebook Account',
+			'wp-module-onboarding'
+		),
+		facebookdesc: __(
```
Comment (officiallygod) [inline src/OnboardingSPA/steps/SiteGen/SocialMedia/contents.js:13]: Let's keep camelCase consistent

File: src/OnboardingSPA/steps/SiteGen/SocialMedia/index.js [hunk 4]
```diff
@@ -22,12 +24,40 @@ const SiteGenSiteSocialMedia = () => {
...
+						<i></i>{ content.facebookbutton }
+						</button>
+					</div>
+				</div>
+				<div className="nfd-onboarding-step--site-gen__social-media__container__buttons">
+					<SkipButton
+						callback={ () => resetSiteLogo() }
```
Comment (officiallygod) [inline src/OnboardingSPA/steps/SiteGen/SocialMedia/index.js:54]: Why do we need this?

File: src/OnboardingSPA/styles/app.scss [hunk 5]
```diff
@@ -66,6 +66,7 @@
 @import "../steps/SiteFeatures/stylesheet";
 @import "../steps/TheFork/stylesheet";
 @import "../steps/SiteGen/SiteLogo/stylesheet";
+@import "../steps/SiteGen/SocialMedia/stylesheet.scss";
```
Comment (officiallygod) [inline src/OnboardingSPA/styles/app.scss:69]: We can omit the extension.

**Review by officiallygod — COMMENTED** [review]
Please do merge the latest [enhance/ai-onboarding](https://github.com/newfold-labs/wp-module-onboarding/tree/enhance/ai-onboarding) branch as I think there are numerous changes.

**Review by diwanshuster — CHANGES_REQUESTED** [review]
Can you check for mobile responsiveness as well?

**Comment by girish-lokapure** [comment]
> Can you check for mobile responsiveness as well?

done

**Review by diwanshuster — APPROVED** [review]
LGTM

**Pull request merged**